components/form/LocationField.tsx

```diff
@@ -72,6 +83,7 @@ export const LocationField = React.forwardRef<RNView, LocationFieldProps>(({name
   const onPress = useCallback(
```

Comment:
I think with the current implementation if you were to clear your location, the `field` would still have the data in there and you could still submit the observation with the previously chosen point. Do you think we could get clearing behavior by updating the form field to be empty?

Reply:
ha, not sure why overcomplicated it so much originally. let me know if this looks more like it to you!